apps/aevm/src/aevm_eeevm_state.erl

```diff
@@ -108,21 +124,24 @@ init(#{ env  := Env
          },
 
     init_vm(State,
-            maps:get(code, Exec),
-            init_storage(Address, Pre)).
-
-init_vm(State, Code, Store) ->
-    State#{ out       => <<>>
-          , call      => #{}
-          , callcreates => []
-          , code      => Code
-          , cp        => 0
-          , logs      => []
-          , memory    => #{}
-          , return_data => <<>>
-          , stack     => []
-          , storage   => Store
-          }.
+            maps:get(code, Exec, #{}),
+            maps:get(mem, Exec, #{mem_size => 0}),
+            maps:get(store, Exec, #{})).
```

Comment:
The store shouldn't be part of `Exec`, it's already part of the `ChainState`. Use `ChainAPI:get_store(ChainState)` here.